qemu/tests/cfg/hv_check_cpu_utilization.cfg

```diff
@@ -18,3 +18,7 @@
     thread_process_cmd = "cat fixed-top-pc-result |grep %s|awk -F ' ' '    {print $9;}'|awk '{sum+=$1} END {print sum/NR}'"
     thread_cpu_level = 5
     post_command = rm -f fixed-top-pc-result
+    variants:
+        - @default:
+        - after_migration:
```

Comment:
I think:
```
- with_migration:
    variants:
        - after_reboot:
          migrate_after_reboot = yes
```
this will be more clear to meet you wanna.
Or simply
```
- with_migration:
   do_mirgation = yes
```

since your `after_migration` maybe tell do other actions after migrating.